polaris-react/src/components/Button/Button.scss

```diff
@@ -155,6 +155,12 @@
     }
```

Comment:
This file's changes are temporary to show what the Banner would look like once the Button updates are merged. When merging this PR, I will revert these changes and change my base to main. This means the Banner will use pre edition buttons until Button is merged